### ISSUE TITLE
Quiescent current from the LED Pixel?

### ISSUE THREAD

**Issue by EternityForest** [tux-friend/tux-cube#1]
As I understand it those chips use 250uA or so when idle?  Seems like a significant load in this kind of application.

Could a MOSFET be used to stop this idle drain?

**Comment by tux-friend** [comment]
This is true. It's my first ESP32 PCB design and I didn't focus on power efficency - but for a revision of the board I sure will take this into consideration.
The sensor cube uses Bluetooth Low Energy and then the ESP32 uses quite a lot of power. For a 24/7 usage of the board the LED could also be soldered of or yes maybe a MOSFET.
